internal/testutil/acc/cluster.go

```diff
@@ -10,14 +10,28 @@ import (
 )
 
 type ClusterRequest struct {
+	ProjectID              string
```

Comment:
Would favour defining as a pointer so it is clear that property is optional, however I see the same applies to all other properties in this struct. Maybe a comment top level of the struct clarifying all properties are optional can be an option as well.

Reply:
i would keep strings (with empty being optional), but agree a comment can help

Reply:
Added to list in https://jira.mongodb.org/browse/CLOUDP-260997